apps/settings/locales/settings.en-US.properties

```diff
@@ -494,8 +494,22 @@ wifi-join-password=Password
 hotspotSettings-header={{hotspotSettings}}
 ssid-name=SSID Network Name
 hotspot-open=open
-hotspot-wpa-psk=WPA (TKIP)
-hotspot-wpa2-psk=WPA2 (AES)
+
+# LOCALIZATION NOTE
+#
+# Please check settings.ar.properties to see the difference first.
```

Comment:
This doesn't help at all. Localizers won't know what this file is, they're aware of the English file and their own.